### PR TITLE
Document limitations of namespace-scoped KUDO resources

### PR DESCRIPTION
**Pull request by nfnt**
**What this PR does / why we need it**:
Document limitations of namespace-scoped KUDO resources

### CODE REVIEW

File: content/docs/what-is-kudo.md
Comment (porridge) [inline content/docs/what-is-kudo.md]:
```suggestion
The _Operator_, _OperatorVersion_, and _Instance_ resources created by KUDO are namespace scoped. These resources can only own resources in the same namespace. As a result, it isn't possible for a single operator to create resources in multiple namespaces.
```

**Review by porridge — APPROVED** [review]
One nit inline. I'd also use `Instance` rather than _Instance_, etc.

File: content/docs/what-is-kudo.md
Comment (zen-dog) [inline content/docs/what-is-kudo.md]:
```suggestion
It is possible for an _Instance_ to create cluster-scoped resources. However, these `resourceOwners` field for such resources is not populated with the _Instance_ reference. Cluster-scoped resources are also **not** automatically cleaned up when the _Instance_ is deleted. Additionally,  they can result in issues when updating or upgrading an _Instance_. [KEP-5](https://github.com/kudobuilder/kudo/blob/master/keps/0005-cluster-resources-for-crds.md) will resolve this limitation.
```

**Review by zen-dog — APPROVED** [review]
🚢 with an addition

**Review by meichstedt — APPROVED** [review]
LGTM once the suggestions are merged

File: content/docs/what-is-kudo.md
Comment (zen-dog) [inline content/docs/what-is-kudo.md]:
```suggestion
All the resources created by an operator _Instance_ are [owned](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents) by it. This way, when the _Instance_ is deleted, the resources are removed too. However, the _Instance_ (as well as  _Operator_, and _OperatorVersion_)  are namespace scoped. Namespaced resources can only own resources in the same namespace. As a result, it isn't possible for a single operator to create resources in multiple namespaces.
```

**Pull request merged**
